.github/patch-release-pull-request.md

```diff
@@ -1,10 +1,11 @@
-This is the patch release pull request for WooCommerce Blocks plugin {{version}}.
+This is the patch release pull request for WooCommerce Blocks plugin `{{version}}`.
```

Comment:
Just noting that there are lots of common items/steps between patch and regular releases (on PR and checklist templates). 

Something to keep in mind when we iterate on the templates - may need to update in two places.

More importantly, the common items across the two may actually be important constant principles of our release process. Personally I'd like to see these principles (rationale/why, what matters) elevated to the markdown doc (maybe we can iterate on this as a team over time).